lib/dal/src/builtins/schema/test_exclusive_schema_starfield.rs

```diff
@@ -0,0 +1,320 @@
+use si_pkg::SchemaSpecData;
+use si_pkg::{
+    ActionFuncSpec, AttrFuncInputSpec, AttrFuncInputSpecKind, FuncArgumentSpec, FuncSpec,
+    FuncSpecBackendKind, FuncSpecBackendResponseType, FuncSpecData, PkgSpec, PropSpec, SchemaSpec,
+    SchemaVariantSpec, SchemaVariantSpecData, SiPkg, SocketSpec, SocketSpecData, SocketSpecKind,
+};
+
+use crate::func::argument::FuncArgumentKind;
+use crate::func::intrinsics::IntrinsicFunc;
+use crate::pkg::import_pkg_from_pkg;
+use crate::{prop::PropPath, ActionKind};
+use crate::{BuiltinsResult, DalContext, PropKind};
+
+pub async fn migrate_test_exclusive_schema_starfield(ctx: &DalContext) -> BuiltinsResult<()> {
```

Comment:
Same exact file, but this sole function is no longer a method off of the `MigrationDriver`